hack/install.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Inspired by - https://github.com/vmware-tanzu/community-edition/blob/main/hack/install.sh
+# Script to install tanzu framework
+# Usage: ./hack/install.sh /path/to/tanzu-framework/core/binary
+
+# set -o errexit
```

Comment:
Was there a reason for this being commented out?

Reply:
I'll fix it in a follow-up PR. Thanks!